### PR TITLE
Crosslink error

### PR DESCRIPTION
**Pull request by IsaiasA1**
# Description
Resolved bug dealing with copying directories over separate partitions (os.rename() being incapable, switch to cmdline "cp")

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|dell/csm#164|

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Powermax  Authorization e2e nightly
    6 scenarios ([32m6 passed[0m)
    51 steps ([32m51 passed[0m)
    9m12.728235059s
    2022/11/18 01:02:12 Volume test finished
    --- PASS: TestVolume (552.76s)

### CODE REVIEW

File: deploy/install.go [hunk 1]
```diff
@@ -318,7 +317,8 @@ func (dp *DeployProcess) CopySidecarProxyToCwd() {
 		return
 	}
 	tgtPath := filepath.Join(wd, sidecarImageTar)
-	if err := osRename(sidecarFilePath, tgtPath); err != nil {
+	cmd := execCommand("cp", "--recursive", sidecarFilePath, tgtPath)
```
Comment (coulof) [inline deploy/install.go:320]: Are you sure we can not do this in go ? Now we are not platform-independent.

Btw, you may want to copy and preserve the permissions `cp -p`

Comment (IsaiasA1) [reply deploy/install.go:320]: I'm a little confused on the word "platform-independent" isn't the 'cp' command in every distribution of linux?

Comment (coulof) [reply deploy/install.go:320]: Yes, but not on Windows.

I have to confess I thought go had a built-in to copy a file but looks like not...

**Review by coulof — APPROVED** [review]
Check if permission preserve is needed.

**Review by alikdell — APPROVED** [review]

File: deploy/install.go (hunk 1 above)
Comment (IsaiasA1) [reply deploy/install.go]: added the -p flag to the execCommand

**Review by alikdell — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Comment by sharmilarama** [comment]
run e2e test

**Comment by IsaiasA1** [comment]
run e2e test

**Comment by IsaiasA1** [comment]
run e2e test

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Comment by IsaiasA1** [comment]
run e2e test

**Pull request merged**